internal/neofs/tree.go

```diff
@@ -274,6 +286,122 @@ func (c *TreeClient) DeleteBucketCORS(ctx context.Context, cnrID *cid.ID) (*oid.
 	return nil, nil
 }
 
+func (c *TreeClient) GetObjectTagging(ctx context.Context, cnrID *cid.ID, objVersion *data.NodeVersion) (map[string]string, error) {
+	tagNode, err := c.getObjectTaggingNode(ctx, cnrID, objVersion)
+	if err != nil {
+		return nil, err
+	}
+
+	if tagNode == nil {
+		return nil, nil
+	}
+
+	delete(tagNode.Meta, isTagKV)
+
+	return tagNode.Meta, nil
+}
+
+func (c *TreeClient) PutObjectTagging(ctx context.Context, cnrID *cid.ID, objVersion *data.NodeVersion, tagSet map[string]string) error {
+	tagNode, err := c.getObjectTaggingNode(ctx, cnrID, objVersion)
+	if err != nil {
+		return err
+	}
+
+	tagSet[isTagKV] = "true"
```

Comment:
What if user provide tag with name the same as `isTagKV` constant value? Consider using `aws:` prefix for `isTagKV` (https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/allocation-tag-restrictions.html)

Reply:
You're right, fixed for object tagging. For bucket tagging we don't use `isTagKV`, we keep bucket tags in a system node.

Reply:
> You're right, fixed for object tagging. For bucket tagging we don't use `isTagKV`, we keep bucket tags in a system node.

Yes, you are right. But  user-defined tags can still conflict with `SystemName: bucket-tagging` attribute. So we have to handle this. We can use some prefix for user-defined tags